pureport.version

```diff
@@ -0,0 +1 @@
+17.0.1+1.0
```

Comment:
It might be better to include the name "pureport" in the build number so that the egg that is built has "pureport" in the name to make it absolutely clear that an egg is our version.

Reply:
Yeah I could do that now that the pbr isn't trying to enforce SemVer.

Reply:
Let's put this change on hold for now.